en/docs/reference/connectors/kafka-connector/kafka-connector-avro-producer-example.md

```diff
@@ -141,18 +269,9 @@ Sample API configuration when the Confluent Schema Registry is secured with basi
                       expression="json-eval($.topic)"
                       scope="default"
                       type="STRING"/>
-            <kafkaTransport.init>
-               <name>Sample_Kafka</name>
-               <bootstrapServers>localhost:9092</bootstrapServers>
-               <keySerializerClass>io.confluent.kafka.serializers.KafkaAvroSerializer</keySerializerClass>
-               <valueSerializerClass>io.confluent.kafka.serializers.KafkaAvroSerializer</valueSerializerClass>
-               <schemaRegistryUrl>http://localhost:8081</schemaRegistryUrl>
-               <maxPoolSize>100</maxPoolSize>
-               <basicAuthCredentialsSource>USER_INFO</basicAuthCredentialsSource>
```

Comment:
We have described `basicAuth` in the next section. Shall we add this `init` as a local entry for clarity?

Reply:
Done with d9c38e1ccfe0de6eaf9375ff83d04a3283e6eb67